src/etc/frontend/routes.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:App/etc/routes.xsd">
     <router id="standard">
-        <route id="hyvaCheckout" frontName="hyva">
-            <module name="Hyva_Checkout" />
+        <route id="react_checkout" frontName="react-checkout">
```

Comment:
I feel the route name does not need to be changed. It should remain `hyva` in my opinion. Because, ideally, any alternative checkout won't work together with Hyva react checkout. Hence route `hyva` feels "Good" and `react-checkout` feels "Huono". For the hyva alternative checkout also, we can use `hyva` as frontend route name so that it will be a good indication that it is a hyvä stuff. What do you think?

Reply:
In theory (and in my case when testing locally), you could have both React and Magewire running but not with an active redirect/override.
It also makes the layout xml files more distinctive

Reply:
Also change the id to `hyva_react_checkout`. We need an indication of Namespace there.

Reply:
Okay, changing it to `hyva` and the controllername to `reactcheckout`.

> `/hyva/reactcheckout/`

Reply:
Sounds good :)

Reply:
Okay, updated